stdlib/src/sys/ffi.mojo

```diff
@@ -317,6 +317,225 @@ struct DLHandle(CollectionElement, CollectionElementNew, Boolable):
 
         return res
 
+    # FIXME
```

Comment:
**Question** What's the bug/issue with using variadics here?  Did you file a GitHub issue?  This is, of course, the nicer way to do it 😄

Reply:
No support for unpacking `VariadicPack` into its types yet. I'll merge with nightly and add the comment

Reply:
Feedback internally would like to see variadic support here in the new `ffi.call` function before proceeding forward.  Is there a way we can achieve this without unpacking support for variadics?

Reply:
I can't believe I actually managed to get it to work. It now unpacks the thing fully, no more overloads needed :) :tada: